### DISCUSSION TITLE
<C-Space> does nothing

### DISCUSSION THREAD

**Discussion by Geobert** [neovide/neovide#2668] [Q&A] [answered]
Hi, 

I’m beginning my journey with nvim through neovide, on Windows (for now).

I’m using `kickstart` and it has `<C-Space>` mapped to trigger `cmp` but it doesn’t work.
I added another map to test the command and it works, so something is messing around with C-space and I can’t see what :-/

**Comment by Geobert** [comment]
nevermind, it’s on nvim side https://github.com/neovim/neovim/issues/8435

**Reply by fredizzimo** [reply]
Sorry, I did not see this until now. But I don't think that bug is relevant for a gui like Neovide. We should detect and send <C-space> correctly.

Can you run `neovide --log`, press <C-space> and then quit. And attach the logfile here?

**Reply by Geobert** [reply]
Sure :)
[neovide_rCURRENT.zip](https://github.com/user-attachments/files/15987088/neovide_rCURRENT.zip)

**Reply by fredizzimo** [reply, marked as answer]
I opened an issue for it here 
* https://github.com/neovide/neovide/issues/2669
